pkg/printers/internalversion/describe.go

```diff
@@ -2650,6 +2653,18 @@ func getPodStatusForController(c coreclient.PodInterface, selector labels.Select
 	return
 }
 
+// isControllerRef returns true if a pod is managed by a controller or if owner
+// reference is not set; otherwise, return false.
```

Comment:
An orphan pod = a pod that doesn't have a controllerRef specified. ControllerRef is a lock when one child is selected by multiple parents via selector. That's to say, a controller can select a pod that it doesn't own, but the controller won't try to manage that pod. 

A pod can be orphaned when its controller is deleted non-cascadingly. Other existing controllers with matching selector will try to adopt it, and eventually at most one controller will win.